src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -672,6 +672,15 @@ internal async Task SendInvocationRequest(ScriptInvocationContext context)
                     return;
                 }
 
+                foreach ((string, DataType, object) element in context.Inputs)
```

Comment:
nit: use deconstruction:

```suggestion
                foreach ((string name, DataType dataType, object binding) in context.Inputs)
```

Then instead of `element.ItemX`, you can use appropriate variable from above.

Reply:
Change here is not needed now